docs/styleguide.config.js

```diff
@@ -201,6 +201,7 @@ module.exports = {
         '../react/BottomNavigation/index.js',
         '../react/BottomNavigationAction/index.js',
         '../react/Box/index.js',
+        '../react/BreadcrumbsMUI/index.js',
```

Comment:
on base maintenant tous nos composants sur Mui, pas la peine de mettre Mui dans le nom de celui-ci.

Par rapport à l'autre on peut soit le déprécier et le déplacer (en faisant un breaking change), soit carrément le remplacer si l'API n'est pas trop différente...

Je regarde plus en détail plus tard

Reply:
Hello 👋 

Here is the PR that asks for a Breadcrumb: https://github.com/cozy/cozy-drive/pull/2454

The related story is: https://trello.com/c/S9F7SMFM/1020-%F0%9F%93%81-drive-bread-crump

I don't have any link to the Figma, how can I get access to it?

I'm a bit lost about what to do. At first, I will follow your recommendation to import Breadcrumb from MUI directly in CozyDrive, make a perfect design, because adding it in CozyUI.

Reply:
After discussing with @Ldoppea , we distinguish several usage of Breadcrumbs in CozyApps: 

- a line Breadcrumbs in Drive / and maybe in the future My CO2 or CozyPass (need to discuss with @joel-costa )
<img width="1100" alt="Capture d’écran 2021-12-23 à 15 16 45" src="https://user-images.githubusercontent.com/8363334/147343439-4322a0e9-6c3d-432f-92e9-bf433761c079.png">


- a compact Breadcrumbs in Banks, see screenshots:
![Image Pasted at 2021-12-24 10-25(1)](https://user-images.githubusercontent.com/8363334/147343469-3e2abcaf-6b5e-4145-9127-21a874248dfc.png)
![Image Pasted at 2021-12-24 10-25](https://user-images.githubusercontent.com/8363334/147343471-9881cf0a-a3f8-41d4-ac1e-4a2d2d28bd82.png)